_data/team-members.yml

```diff
@@ -194,10 +194,12 @@ members:
     twitter: adrienpi2
     photo: /resources/img/adrien.png
     currentlyWorkingOn: true
-    contact_for: [scaladex]
+    contact_for: [scala3MigrationGuide]
     projects:
+        - title: scala3MigrationGuide
+          percentage: 90
         - title: scaladex
-          percentage: 100
+          percentage: 10
```

Comment:
Should we also add BSP implementation in sbt?

Reply:
done